.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   validate:
     docker:
-      - image: trussworks/circleci-docker-primary:a18ba9987556eec2e48354848a3c9fb4d5b69ac8
+      - image: trussworks/circleci-docker-primary:c3cdcb24c00afc1ba635f8f6efd215f55aaf2079
```

Comment:
Can you bump this to `22977732ef16c3599f139654fdae7c7df8335dff` that will include the new terraform-docs release.